.circleci/config.yml

```diff
@@ -188,6 +188,26 @@ jobs:
             ( echo "Please commit an up to date Cargo.lock"; exit 1 )
       - save-sccache
       - save-cargo-cache
+  cargo-audit:
+    executor: docker-rust
+    resource_class: small
+    steps:
+      - checkout
+      - run: |
+          echo "$(cargo search cargo-audit | grep '^cargo-audit =' | sha256sum)" > /tmp/cargo-audit-cache-key
+      - restore_cache:
+          name: Restore Cargo audit cache
+          keys: 
+            - cargo-audit-{{ checksum "/tmp/cargo-audit-cache-key" }}
+      - run: |
+          cargo install --force cargo-audit
```

Comment:
If we download a binary instead, we don't need caching, see for example how sccache is done